src/providers/export/exportProvider.test.ts

```diff
@@ -10,7 +10,11 @@ describe("Export Provider Base", () => {
     it("initializes the asset and storage providers", () => {
         registerProviders();
 
-        ExportProviderFactory.register("test", (project) => new TestExportProvider(project));
+        ExportProviderFactory.register({
+            name: "test",
+            displayName: "Test DisplayNameq",
```

Comment:
Nit: typo with "q" at the end.

Reply:
I almost changed this to Test DisplayName without Q, but alas, just removed the q.